source/docs/api-reference/wpilib-integration/motorcontroller-integration.rst

```diff
@@ -1,19 +1,7 @@
 MotorController Integration
 ===========================
 
-Phoenix Pro motor classes such as ``TalonFX`` (`Java <https://api.ctr-electronics.com/phoenixpro/release/java/com/ctre/phoenixpro/hardware/TalonFX.html>`__, `C++ <https://api.ctr-electronics.com/phoenixpro/release/cpp/classctre_1_1phoenixpro_1_1hardware_1_1_talon_f_x.html>`__) implement the ``MotorController`` (`Java <https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/wpilibj/motorcontrol/MotorController.html>`__, `C++ <https://github.wpilib.org/allwpilib/docs/release/cpp/classfrc_1_1_motor_controller.html>`__) interface.
-
-Phoenix Pro implements all inherited functions:
-
-- ``set``
-- ``setVoltage``
-- ``get``
-- ``setInverted``
-- ``getInverted``
-- ``disable``
-- ``stopMotor``
-
-Due to inheriting the ``MotorController`` interface, this allows motors such as ``TalonFX`` to be included as part of the ``DifferentialDrive`` constructor.
+Phoenix Pro motor controller classes such as ``TalonFX`` (`Java <https://api.ctr-electronics.com/phoenixpro/release/java/com/ctre/phoenixpro/hardware/TalonFX.html>`__, `C++ <https://api.ctr-electronics.com/phoenixpro/release/cpp/classctre_1_1phoenixpro_1_1hardware_1_1_talon_f_x.html>`__) implement the ``MotorController`` (`Java <https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/wpilibj/motorcontrol/MotorController.html>`__, `C++ <https://github.wpilib.org/allwpilib/docs/release/cpp/classfrc_1_1_motor_controller.html>`__) interface. This allows motor controllers such as ``TalonFX`` to be included as part of the ``DifferentialDrive`` constructor.
```

Comment:
For the last sentence,
"This allows Phoenix Pro motor controllers to be used in WPILib drivetrain classes such as ``DifferentialDrive``.